doc/test.md

```diff
@@ -0,0 +1,54 @@
+RFC2223-JP.txt 2000/09/27 (initial version)
+(C)JPNIC RFC-JP project, 2000. All rights reserved.
+
+
+
+
+Network Working Group                                          J. Postel
+Request for Comments: 2223                                   J. Reynolds
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Error - Line 8 sentence length(154) exceeds the maximum sentence length of 100.